### PR TITLE
ZEPPELIN-331 Don't update back the browser where updated the angular object

### PR DESCRIPTION
**Pull request by Leemoonsoo**
This PR fixes problem described in 
https://issues.apache.org/jira/browse/ZEPPELIN-331

By excluding the webbrowser who created the angular object update event from broadcast target.

* [x] exclude from broadcast
* [x] add unittest

### CODE REVIEW

**Comment by corneadoug** [comment]
Do you know how to reproduce that issue?

**Comment by Leemoonsoo** [comment]
You can easily reproduce when there are big network latency between your browser and zeppelin server.

**Comment by Leemoonsoo** [comment]
Ready to merge

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 1]
```diff
@@ -261,6 +261,26 @@ private void broadcast(String noteId, Message m) {
     }
   }
 
+  private void broadcastExcept(String noteId, Message m, NotebookSocket exclude) {
+    synchronized (noteSocketMap) {
+      List<NotebookSocket> socketLists = noteSocketMap.get(noteId);
+      if (socketLists == null || socketLists.size() == 0) {
+        return;
+      }
+      LOG.info("SEND >> " + m.op);
```
Comment (jongyoul) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: You'd better change the level of this log to debug. I saw the issue to change it with this kind of logs

Comment (Leemoonsoo) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: Thanks for finding this one. update pushed.

**Pull request closed without merging**